payloads/import_export.go

```diff
@@ -28,11 +31,48 @@ type ImportRequestPayload struct {
 	// Otherwise the server SHALL store the object as provided.
 	KeyWrapType kmip.KeyWrapType `ttlv:",omitempty"`
 	// All of the object’s Attributes.
-	Attribute []kmip.Attribute `ttlv:",omitempty"`
+	Attribute []kmip.Attribute
 	// The object value being imported, in the same manner as the Register operation.
 	Object kmip.Object
 }
 
+func (pl *ImportRequestPayload) TagDecodeTTLV(d *ttlv.Decoder, tag int) error {
+	return d.Struct(tag, func(d *ttlv.Decoder) error {
+		if err := d.TagAny(kmip.TagUniqueIdentifier, &pl.UniqueIdentifier); err != nil {
+			return err
+		}
+
+		if d.Tag() == kmip.TagReplaceExisting {
```

Comment:
Suggestion, you can use `d.Opt(kmip.TagReplaceExisting, &pl.ReplaceExisting)` to reduce the boilerplate


```go
func (dec *Decoder) Opt(tag int, value any) error {
	if dec.Tag() == tag {
		return dec.TagAny(tag, value)
	}
	return nil
}
```